common/src/main/java/com/wynntils/core/text/StyledText.java

```diff
@@ -105,6 +107,10 @@ public static StyledText fromComponent(Component component) {
         return new StyledText(parts, temporaryWorkaround, clickEvents, hoverEvents);
     }
 
+    public static StyledText fromComponent(Component component, boolean onlyCodes) {
+        return onlyCodes ? fromString(component.getString()) : fromComponent(component);
+    }
+
```

Comment:
What is this and why is this needed? I don't event think this properly works. Either way, we should not need to use this and if there are places where we depend on only using color codes from components and not actual styles, then we need to update those places to be correct.

Reply:
this is the replacement of fromComponentIgnoringComponentStylesAndJustUsingFormattingCodes as it has such a long name I thought it must be very important ^^. i just copied the behavior to not introduce new bugs.